plugins/restful/RestfulDataProviderEFQ.php

```diff
@@ -140,7 +140,9 @@ protected function queryForListSort(\EntityFieldQuery $query) {
 
     foreach ($sorts as $public_field_name => $direction) {
       // Determine if sorting is by field or property.
-      $property_name = $public_fields[$public_field_name]['property'];
+      if (!$property_name = $public_fields[$public_field_name]['property']) {
```

Comment:
I'm not sure I follow. `getColumnFromProperty` will be run later. This check only makes sure that there is a `'property'` key in the public field definition. This is the same thing we were doing before, but now it's wrapped in an if clause.